packages/ui-patterns/src/MetricsCard/index.tsx

```diff
@@ -0,0 +1,267 @@
+  )
+)
+MetricsCardIcon.displayName = 'MetricsCardIcon'
+
+interface MetricsCardLabelProps extends React.HTMLAttributes<HTMLDivElement> {
+  tooltip?: string
+  children: React.ReactNode
+}
+
+const MetricsCardLabel = React.forwardRef<HTMLDivElement, MetricsCardLabelProps>(
+  ({ className, tooltip, children, ...props }, ref) => {
+    return (
+      <CardTitle
+        ref={ref}
+        className={cn('flex items-center gap-2 text-foreground-light', className)}
+        {...props}
+      >
+        <span>{children}</span>
+        {tooltip && (
+          <Tooltip>
+            <TooltipTrigger asChild>
+              <HelpCircle size={14} strokeWidth={1.5} />
+            </TooltipTrigger>
+            <TooltipContent>{tooltip}</TooltipContent>
+          </Tooltip>
+        )}
+      </CardTitle>
+    )
+  }
+)
+MetricsCardLabel.displayName = 'MetricsCardLabel'
+
+const MetricsCardValue = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDivElement>>(
+  ({ className, ...props }, ref) => {
+    const { isLoading } = useMetricsCard()
+
+    if (isLoading) {
+      return <Skeleton className="w-32 h-7" />
+    }
+
+    return (
+      <span ref={ref} className={cn('font-normal text-xl tabular-nums', className)} {...props} />
+    )
+  }
+)
+
+MetricsCardValue.displayName = 'MetricsCardValue'
+
+interface MetricsCardDifferentialProps extends React.HTMLAttributes<HTMLDivElement> {
+  variant?: 'positive' | 'negative' | 'default'
+}
+
+const MetricsCardDifferential = React.forwardRef<HTMLDivElement, MetricsCardDifferentialProps>(
+  ({ className, variant = 'default', ...props }, ref) => {
+    const { isLoading } = useMetricsCard()
+
+    if (isLoading) {
+      return <Skeleton className="w-16 h-5" />
+    }
+
+    return (
+      <span
+        ref={ref}
+        className={cn(
+          variant === 'positive'
+            ? 'text-brand'
+            : variant === 'negative'
+              ? 'text-destructive'
+              : 'text-foreground-light',
+          'tabular-nums text-sm',
+          className
+        )}
+        {...props}
+      />
+    )
+  }
+)
+
+MetricsCardDifferential.displayName = 'MetricsCardDifferential'
+
+const SparklineTooltip = ({ active, payload, label }: RechartsTooltipProps<any, any>) => {
+  if (!active || !payload || !payload.length) return null
+
+  const formatTimestamp = (timestamp: string) => {
+    const date = dayjs(timestamp)
+    const hour = date.hour()
+    const period = hour >= 12 ? 'pm' : 'am'
+    const displayHour = hour % 12 || 12
+
+    return `${date.format('MMM D')}, ${displayHour}${period}`
+  }
+
+  return (
+    <div className="bg-black/90 text-white p-2 rounded text-xs">
+      {label && (
+        <div className="text-foreground-light">{formatTimestamp(payload[0].payload.timestamp)}</div>
+      )}
+      <div>{payload[0].value.toLocaleString(undefined, { maximumFractionDigits: 0 })}</div>
+    </div>
```

Comment:
This top line of text is a bit hard to read on light mode: 

<img width="314" height="218" alt="Image" src="https://github.com/user-attachments/assets/4df68e85-de3f-43b7-b972-b76002622ad7" />

The hardcoded black background and dynamic `foreground-light` combo is probably to blame. How about something like this?


```suggestion
    <div className="bg-black/90 p-2 rounded text-xs">
      {label && <p className="text-white/50">{formatTimestamp(payload[0].payload.timestamp)}</p>}
      <p className="text-white">
        {payload[0].value.toLocaleString(undefined, { maximumFractionDigits: 0 })}
      </p>
    </div>
```

Cleans up a bit of the syntax too. Looks like this:

<img width="264" height="194" alt="Image" src="https://github.com/user-attachments/assets/a14f8473-f14d-4b78-b34f-8899bcce0a0b" />

This is inherited from a few other components, right? I see similar markup in:

- AreaChart
- BarChart